index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
+</head>
+<body>
+    <header class="navigation">
+        <nav class="header">
+            <a class="logo" href="#">My Logo</a>
+            <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
+        </nav>
+    </header>
+
+    <section class="bio-section">
+        <div class="bio">
+            <h1 class="bio-header">I'm Alexandra Glad to see you!</h1>
```

Comment:
- [ ] Kindly adjust the font size of your `h1`main title make the text align ,so that it will look exact like the figma design and adjust the width to make it responsive on mobile view.

- [ ] Figma design
![Screenshot 2023-06-01 172333hh](https://github.com/josman227/Mobile-Portfolio/assets/92458236/d3d0fc5e-34b2-4da5-ad0a-88a828ed6cab)

- [ ] yor design
![Screenshot 2023-06-01 171039](https://github.com/josman227/Mobile-Portfolio/assets/92458236/7ef767ad-b065-4e32-85b1-0bbe2194efc3)